fix(rum): listen to all page clicks (#1219)
elastic/apm-agent-rum-js: docs/configuration.asciidoc

* `page-load`
* `history`
* `eventtarget`
* `click`
* `xmlhttprequest`
* `fetch`
* `error`

NOTE: To disable all `http-request` transactions, add both `fetch` and `xmlhttprequest`.
to this config.

NOTE: To disable `user-interaction` transactions,  add `eventtarget` or `click` to this config.

> The option `eventtarget` is deprecated and it might be removed in a future release.

Author: I have updated the documentation informing that `eventtarget` is deprecated and that might be removed in the future.

The screenshot below shows how it will look like:
<img width="1001" alt="Screenshot 2022-05-12 at 15 18 54" src="https://user-images.githubusercontent.com/15065076/168083998-00482a9a-e9b0-4f16-b37e-c6fbae92f900.png">

What do you think?

Reviewer: Added a small suggestion, looks good 👍🏽

> The option `eventtarget` is deprecated and it might be removed in a future release.

Reviewer: ```suggestion
The option `eventtarget` is deprecated and will be removed in the future releases.
```
